CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(libNEGF VERSION 0.1 LANGUAGES Fortran C)
+
+set(FYPP "fypp" CACHE STRING "Fypp preprocessor")
```

Comment:
I would add a fallback option to use the one in ext_fypp, like
`find_program(FYPP fypp PATHS "${CMAKE_SOURCE_DIR}/ext_fypp/bin/")`

Reply:
Absolutely! I overlooked, that you ship a Fypp with your library already.